src/shapes/object.class.js

```diff
@@ -573,6 +573,14 @@
     selectable:               true,
 
     /**
+     * When set to `false`, an object can not be selected for modification using group-based selection.
+     * But events still fire on it.
+     * @type Boolean
```

Comment:
what do you mean by "but events still fire on it"? If there is some conflict between .selectable and .evented should be solved anyway regardless of this new feature